_posts/2022-12-14-#26631.md

```diff
@@ -122,9 +122,273 @@ is just its physical size, but an input typically
    Bonus: is there a way to create such an output script where this unspendability
    can even be mathematically proven?
 
- <!-- TODO: After meeting, uncomment and add meeting log between the irc tags
  ## Meeting Log
```

Comment:
Looks like the leading whitespace is messing up with md formatting:
<img width="363" alt="image" src="https://user-images.githubusercontent.com/69010457/207824978-d465c9f9-3564-40bd-b734-e39056848973.png">

```suggestion
## Meeting Log
```